.github/workflows/rust.yml

```diff
@@ -26,12 +25,63 @@ jobs:
       uses: arduino/setup-protoc@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-    - name: Run tests
-      run: cargo test --all
-
-#   build:
-#     runs-on: ubuntu-latest
-#     steps:
-#     - uses: actions/checkout@v3
-#     - name: Build
-#       run: docker build . --tag=qdrant
+    - name: Install nextest
+      uses: taiki-e/install-action@nextest
+    - name: Build and run tests
+      # Profile "ci" is configured in .config/nextest.toml
+      run: cargo nextest run --workspace --profile ci
+    - name: Upload test report
+      uses: actions/upload-artifact@v2
+      with:
+        name: junit-${{ matrix.os }}.xml
+        path: target/nextest/ci/junit.xml
```

Comment:
Is this is a proper and full replacement for `cargo test --all`? Or do we better create another job just for nextest runs?

Reply:
Yes, actually, the `--all` flag has been deprecated in favor of `--workspace`. It is full replacement 👍